airgun/entities/flatpak.py

```diff
@@ -83,6 +83,52 @@ def create(self, values):
         view = FlatpakRemoteDetailsView(self.browser)
         view.wait_displayed(delay=3)
 
+    def create_redhat_remote(self, values):
+        """Create a Red Hat Flatpak remote using the info alert action.
+
+        This method clicks the 'Add Red Hat flatpak remote' button from the
+        info alert instead of manually filling the URL.
+
+        :param dict values: values to create the remote with
+        """
+        view = self.navigate_to(self, 'All')
+        view.wait_displayed()
+        view.create_new_btn.click()
+        create_modal = CreateFlatpakRemoteModal(self.browser)
+        create_modal.wait_displayed()
```

Comment:
I would suggest moving all this logic to a 'Create' Navigator, and replacing existing usage of this construct with that instead. Should make it more readable, and reusable.
```suggestion
    view = self.navigate_to(self, 'Create')
```